docs/howto/manage-charms.rst

```diff
@@ -250,6 +251,34 @@ that at present this must be a Discourse page. E.g.,
 
     See more: :ref:`charmcraft-yaml-key-links`
 
+The Description tab can contain additional documentation, such as tutorials or 
+how-to guides.
+
+.. note::
```

Comment:
I like the update! I think it flows well.

Do you think we need the note block? I think it could work as a regular paragraph.

Reply:
Yes, this fragment is important enough to use the note block.
Given that we are in a How-to guide, it feels too generic to put it in the middle of more practical instructions as just usual text.